vscode/package.json

```diff
@@ -249,10 +249,13 @@
     ]
   },
   "scripts": {
-    "tsc:check": "node ../node_modules/typescript/bin/tsc -p ./tsconfig.json",
     "build": "npm run tsc:check && node build.mjs",
     "build:watch": "node build.mjs --watch",
-    "run:web": "npx @vscode/test-web --extensionDevelopmentPath . ../samples"
+    "run:web": "npm exec --no @vscode/test-web --extensionDevelopmentPath . ../samples",
+    "start": "npm exec --no @vscode/test-web --extensionDevelopmentPath . ../samples",
+    "test": "node ./test/runTests.mjs",
+    "pretest": "npm exec --no playwright install && node ./test/buildTests.mjs",
```

Comment:
So is this the only time the tests get built or checked - when the tests are run?

Reply:
Yes. I can add `buildTests.mjs` to the regular build, but I didn't see a need to. 

They do get checked by eslint. But not by TypeScript, at least not in the build. You get TypeScript errors in the editor of course.

I can add in the checking steps for the test code, but again, I didn't feel it necessary. (I can be talked into it).